input/resources/au-core-responder.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <CapabilityStatement xmlns="http://hl7.org/fhir">
-  <id value="au-core-server"/>
+  <id value="au-core-responder"/>
   <text>
     <status value="extensions"/>
     <div xmlns="http://www.w3.org/1999/xhtml">
-      <h2 id="title">AU Core Server CapabilityStatement</h2>
+      <h2 id="title">AU Core Responder CapabilityStatement</h2>
       <ul>
         <li>Implementation Guide Version: 0.3.0-ci-build</li>
```

Comment:
Incorrect version number; should be 0.4.0-ci-build or left as TBD.

Reply:
Not in scope of this PR.